src/components/Authentication/SignUp.js

```diff
@@ -57,11 +57,23 @@ const SignUpFormBase = props => {
     props.firebase
       .doCreateUserWithEmailAndPassword(form.email, form.passwordOne)
       .then(authUser => {
+        authUser.user
+          .updateProfile({
+            displayName: form.username,
+            photoURL:
+              'https://firebasestorage.googleapis.com/v0/b/bookio.appspot.com/o/images%2Frobot.png?alt=media&token=d0cc8a48-aaad-4e3b-9c08-3e341ed8165e',
```

Comment:
Should this URL be hard-coded in here? If so, maybe define it as some sort of const or macro at the top of the file.

Reply:
That's good. I just uploaded a default picture for all new users.